app/assets/javascripts/code/services/_sensors.js

```diff
@@ -1,5 +1,8 @@
 import _ from 'underscore';
 
+const ALL_SENSOR = { id: "all", select_label: "All", session_count: 0 };
+const ALL_PARAMETER = { id: "all", label: "All" };
```

Comment:
I'm not sure if all_sensor would be understandable for someone reading this in the future and not familiar with the details of this problem. How about `ALL_SENSORS_ENTRY` or something like that?

Reply:
I agree. But for now I'm leaving like this. I feel the issue should be solved at a different level. Hopefully will come back here soon